test/unit/app/controllers/transactions/tx-state-manager-test.js

```diff
@@ -33,7 +33,6 @@ describe('TransactionStateManager', function () {
     it('should emit a signed event to signal the exciton of callback', function (done) {
       const tx = { id: 1, status: 'unapproved', metamaskNetworkId: currentNetworkId, txParams: {} }
       const noop = function () {
-        assert(true, 'event listener has been triggered and noop executed')
```

Comment:
Nit: This seems fine for now, but ideally we'd have at least one assertion in each test. This test also doesn't fail very gracefully - the error message you get when `done` isn't called isn't the most helpful.

Maybe it would be better to assert that this was called using a spy? I don't think this test needs to be async either.

The same applies for the `should emit a rejected event to signal the exciton of callback` test below as well, where an `assert(true` was also removed.